includes/admin/settings/views/html-admin-page-shipping-zones.php

```diff
@@ -37,9 +37,14 @@
 					uasort( $methods, 'wc_shipping_zone_method_order_uasort_comparison' );
 
 					if ( ! empty( $methods ) ) {
-						foreach ( $methods as $method ) {
+						foreach ( $methods as $instance_id => $method ) {
 							$class_name = 'yes' === $method->enabled ? 'method_enabled' : 'method_disabled';
-							echo '<li class="wc-shipping-zone-method ' . esc_attr( $class_name ) . '">' . esc_html( $method->get_title() ) . '</li>';
+
+							$shipping_method_link_or_title = esc_html( $method->get_title() );
+							if ( $method->has_settings() ) {
+								$shipping_method_link_or_title = '<a href="admin.php?page=wc-settings&amp;tab=shipping&amp;instance_id=' . $instance_id . '">' . $shipping_method_link_or_title . '</a>';
```

Comment:
@seostudio Thanks for submitting the PR. Could you use `admin_url()` to generate this link and ensure the URL is properly escaped?

Reply:
Fixed.